utils/configuration.go

```diff
@@ -76,7 +79,52 @@ func ParseServerTLS(configuration *viper.Viper, tlsRequired bool) (*tls.Config,
 		}
 	}
 
-	return tlsconfig.Server(tlsOpts)
+	// Fetch TLS config as usual
+	conf, err := tlsconfig.Server(tlsOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	// Reload certificate hourly
+	var reloadCert *tls.Certificate
+	var m sync.RWMutex
+	var resetCert func()
+	resetCert = func() {
```

Comment:
no need for line 91. `resetCert := func() {` is sufficient

Reply:
It's necessary in this case because we call the time.AfterFunc bit inside the declaration